lib/pharos/phases/configure_host.rb

```diff
@@ -35,7 +39,8 @@ def call
           exec_script(
             'configure-docker.sh',
             DOCKER_PACKAGE: 'docker.io',
-            DOCKER_VERSION: "#{Pharos::DOCKER_VERSION}-0ubuntu1~16.04.2"
+            DOCKER_VERSION: "#{Pharos::DOCKER_VERSION}-0ubuntu1~16.04.2",
+            HTTP_PROXY: @host.http_proxy.to_s
```

Comment:
These seem to be everywhere. `Host` could have something like `default_env` that would be merged in `exec_script`

Reply:
I'm not sure if we actually need these anymore. `HTTP_PROXY` should be already set via `/etc/environment`.